### DISCUSSION TITLE
自制mosdns v4使用教程（Win）

### DISCUSSION THREAD

**Discussion by hoicau** [IrineSistiana/mosdns#449] [Show and tell]
项目地址：https://github.com/c2xvi/easier-mosdns
也请开发者大佬收录一下哦

**Comment by liangxyz** [comment]
看了下，本地监听端口1053，请问如何本机使用这个dns服务器呢？

**Reply by hoicau** [reply]
如果需要监听本机请改为53，监听1053是为了给v2rayN这样的代理软件使用的

**Comment by Zerorigin** [comment]
感谢分享，刚从 3.X 升级到 4.X，某些配置不兼容旧版本了，节约了不少查阅官方文档的时间。
